internal/jsonext/jsonext_test.go

```diff
@@ -0,0 +1,44 @@
+package jsonext_test
+
+import (
+	"encoding/json"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/kopia/kopia/internal/jsonext"
+)
+
+type MyStruct struct {
+	Timeout jsonext.Duration `json:"timeout"`
+}
+
+func TestDurationJSONMarshaling(t *testing.T) {
+	ms := MyStruct{Timeout: jsonext.Duration{20*time.Minute + 10*time.Second}}
+
+	b, err := json.Marshal(ms)
+	require.NoError(t, err)
+	require.Equal(t, `{"timeout":"20m10s"}`, string(b))
+}
+
+func TestDurationJSONUnmarshaling(t *testing.T) {
```

Comment:
can you also test that plain number parses correctly as nanoseconds (for compatibility with regular duration stored as a number)?

Reply:
Good catch. Addressed.
@jkowalski PTAL